netdata.spec.in

```diff
@@ -47,18 +47,25 @@ make %{?_smp_mflags}
 rm -rf "%{buildroot}"
 make %{?_smp_mflags} install DESTDIR="%{buildroot}"
 find "%{buildroot}" -name .keep -exec rm {} \;
+touch %{buildroot}/%{_sysconfdir}/netdata/netdata.conf
+%if 0%{?fedora} || 0%{?rhel} >= 7 
+install -m 0644 -D system/netdata-systemd ${RPM_BUILD_ROOT}%{_unitdir}/netdata.service
+%endif
 
 %files
 %attr(-, netdata, netdata) %dir %{_localstatedir}/cache/%{name}/
 %attr(-, netdata, netdata) %dir %{_localstatedir}/log/%{name}/
+%attr(-, netdata, netdata) %{_datadir}/%{name}/
 %config(noreplace) %{_sysconfdir}/%{name}/
 %if 0%{?rhel}
 %{_sbindir}/%{name}
 %else
 %{_bindir}/%{name}
 %endif
-%{_datadir}/%{name}/
```

Comment:
Not sure we are talking about the same thing here. I was referring to line 58 on the right side, in the full patch, which is not shown in the snippet above (which starts at 59).  It sets the ownership of /usr/share/netdata to netdata:netdata.  This is required by the application when running as user netdata.  If it's owned by root, the application will log an error and refuse to read the dir, even though permissions permit read access:

16-03-31 07:33:39: ERROR: netdata: 1: File '/usr/share/netdata/web/' is owned by user 0 (I run as user 498). Access Denied.

I only added the if/else for rhel on bindir/sbindir because that's what I was fixing at the time.  That should be the case for both fedora and rhel, though. I thought about removing sbindir entirely, but wasn't sure if it might break other rpm-based distros like SUSE, so if %if'd it instead.